l10n_be_cooperator_national_number/models/company.py

```diff
@@ -23,6 +23,18 @@ def _check_national_number(self):
                     )
                 )
 
+    def get_display_national_number(self, is_company):
+        self.ensure_one()
+        if is_company:
+            return False
+        return self.display_national_number
+
+    def get_require_national_number(self, is_company):
+        self.ensure_one()
+        if is_company:
+            return False
+        return self.require_national_number
```

Comment:
these already exist on `subscription.request`. it makes sense to move them to `res.company`, but `subscription.request` should be changed to avoid code duplication.

likewise, `l10n_be_cooperator_website_national_number` also uses the same checks. if it moves to `res.company`, the code there should also be removed. edit: oh, this is done below, great!

Reply:
moved to company